api/v1beta1/awscluster_webhook_test.go

```diff
@@ -368,6 +360,56 @@ func TestAWSCluster_ValidateUpdate(t *testing.T) {
 			},
 			wantErr: true,
 		},
+		{
+			name: "Should fail if controlPlaneLoadBalancer healthcheckprotocol is updated",
```

Comment:
What happens if during update we don't pass anything in `HealthCheckProtocol ` ? Might add a case mentioning this scenario is not possible

Reply:
Since we are not using defaukting webhooks and handling the default value in loadbalancer code itslef, have added the test for default case there itself.